app/src/registrationDirective.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    angular
+        .module('webshopModule')
+        .directive('passwordMatch', function () {
+            return {
+                restrict: 'A',
+                scope: true,
+                require: 'ngModel',
+                link: function (scope, elem, attrs, control) {
+                    var checker = function () {
+                        var pwd1 = scope.$eval(attrs.ngModel);
```

Comment:
Yeah, pwd1 works with `control.$modelValue`, too. Should I commit and push the modfied file to my remote branch?